website/project/forms.py

```diff
@@ -1,19 +1,17 @@
 # -*- coding: utf-8 -*-
-from framework.forms import Form, TextField, TextAreaField, validators
-from framework.forms import (
-    BootstrapTextArea,
-    BootstrapTextInput,
-)
+from framework.forms import Form, TextField, validators
 
 ###############################################################################
 # Forms
 ###############################################################################
 
+
 class NewNodeForm(Form):
     title = TextField('Title', [
         validators.Required(message=u'Title is required'),
         validators.Length(min=1, message=u'Title must contain at least 1 character.'),
         validators.Length(max=200, message=u'Title must contain fewer than 200 characters.')
-    ], widget=BootstrapTextInput())
-    description = TextAreaField('Description', widget=BootstrapTextArea())
-    category = TextAreaField('Category', widget=BootstrapTextArea())
+    ])
+    description = TextField('Description')
+    category = TextField('Category')
+    inheritContributors = TextField('Inherit')
```

Comment:
Also, looking at the other WTForms in the codebase, e.g. https://github.com/CenterForOpenScience/osf.io/blob/65aa012f20c3e3e099bc1a09241a231b8a316501/framework/auth/forms.py#L193, it looks like we follow `lower_camel_case` for field names.